src/puppetlabs/puppetdb/cli/shovel.clj

```diff
@@ -0,0 +1,374 @@
+(ns puppetlabs.puppetdb.cli.shovel
```

Comment:
Might also be worth noting that this code also "upgrades" all commands to the current POST-with-headers style, so that now, all other PDB code can assume it will only see the latest format.